Formula/mtn.rb

```diff
@@ -0,0 +1,29 @@
+class Mtn < Formula
+  desc "Saves thumbnails (screenshots) of movie or video files to jpeg files"
+  homepage "https://gitlab.com/movie_thumbnailer/mtn/wikis/home"
+  url "https://gitlab.com/movie_thumbnailer/mtn/-/archive/3.3.2/mtn-3.3.2.tar.gz"
+  sha256 "471ec0172a1753d684032edb8296ef01af115538619daa417678ac037328daf5"
+
+  depends_on "ffmpeg"
+  depends_on "gd"
+
+  def install
+    system "make", "-Csrc", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    # `test do` will create, run in and delete a temporary directory.
+    #
+    # This test will fail and we won't accept that! For Homebrew/homebrew-core
+    # this will need to be a test that verifies the functionality of the
+    # software. Run the test with `brew test mtn`. Options passed
+    # to `brew install` such as `--HEAD` also need to be provided to `brew test`.
+    #
+    # The installed folder is not in the path, so use the entire path to any
+    # executables being tested: `system "#{bin}/program", "do", "something"`.
```

Comment:
Can you please remove the template comments?

Also, if possible, please squash all your commits so that you have only one commit with the message of your first commit: `mtn 3.3.2 (new formula)`

Reply:
Done.